app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/practice/CardViewModel.kt

```diff
@@ -109,7 +109,7 @@ class CardViewModel(
             materialsQueue.add(material.data)
         }
 
-        require(material.data is Symbol)
+        require(material.data is Symbol) // TODO #250 change for symbols desk

```

Comment:
@winter-yuki  may you please fix the "Special Symbols" deck (consists of markers) in Practice mode?

Reply:
Yes, of course, I am going to do it a bit later